components/raftstore-v2/src/operation/command/mod.rs

```diff
@@ -320,11 +320,10 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
     }
 
     pub fn on_apply_res<T>(&mut self, ctx: &mut StoreContext<EK, ER, T>, apply_res: ApplyRes) {
-        if !self.serving() {
-            return;
+        if !self.serving() || !apply_res.admin_result.is_empty() {
+            // TODO: remove following log once stable.
+            info!(self.logger, "on_apply_res"; "apply_res" => ?apply_res, "apply_trace" => ?self.storage().apply_trace());
```

Comment:
Maybe the log information can be tuned to better reflect what it means.

Reply:
The log is a debug log to show the API is called.

Reply:
> The log is a debug log to show the API is called.

But now it's not printed each time on_apply_res is called. It will only print when  !self.serving() || !apply_res.admin_result.is_empty(). But the log content seems that it printed whenever on_apply_res is called.

Reply:
The condition is just to reduce frequency. What's your suggestion?